tests/comparisons/Diff/addremove/the_diff_add_remove_mysql.php

```diff
@@ -35,4 +35,3 @@ public function down()
             ->update();
     }
 }
-
```

Comment:
All the modifications to the "comparisons" (this one and below) are going to break the tests as they are probably expected in the comparisons.

Reply:
Sorry, I run the fixer on all folders. I’ll revert those changes.

Reply:
@robertpustulka This is probably what's currently breaking the MySQL builds.

Reply:
@HavokInspiration The changes to comparison files is just whitespace change. I doubt that's what causing this error https://travis-ci.org/cakephp/migrations/jobs/288989818#L531

Reply:
Those whitespaces were important it seems :)

Reply:
@ADmad The real error is here : https://travis-ci.org/cakephp/migrations/jobs/288989818#L558-L565

What you highlighted was just a side-effect of a previous test failing.